### DISCUSSION TITLE
Rename "search for quests here" option

### DISCUSSION THREAD

**Discussion by FloEdelmann** [streetcomplete/StreetComplete#2434] [Q&A] [answered]
Now that the tiles are downloaded and cached together with quests (starting in v29.0-beta1), I'd propose to rename the "search for quests here" menu option to "download quests and map here".

**Comment by westnordost** [comment, marked as answer]
The idea is that users should not have to worry at all about map not being available when offline anymore to the point where it doesn't need to be mentioned.
